Playground/RxSwiftExtPlayground.playground/Pages/nwise.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,53 @@
+/*:
+ > # IMPORTANT: To use `RxSwiftExtPlayground.playground`, please:
+ 
+ 1. Make sure you have [Carthage](https://github.com/Carthage/Carthage) installed
+ 1. Fetch Carthage dependencies from shell: `carthage bootstrap --platform ios`
+ 1. Build scheme `RxSwiftExtPlayground` scheme for a simulator target
+ 1. Choose `View > Show Debug Area`
+ */
+
+//: [Previous](@previous)
+import RxSwift
+import RxSwiftExt
+
+/*:
+ ## nwise
+
+ The `nwise` operator groups elements emitted by an Observable into arrays, where each array consists of the last N consecutive items; similar to a sliding window.
```

Comment:
The term "last" is confusing here. can you explain what you meant? 
Seems to me like its "each array consists of N consecutive items"

Reply:
To be honest, I haven't figured out the best way to phrase something so visually trivial. This is why I have three similar, yet different versions in all places...

Reply:
What about just simplifying it as:
> The `nwise` operator groups elements emitted by an Observable into arrays of `n` items each